### PR TITLE
fix: add namespace flag for nydusify commit.

### PR DESCRIPTION
**Pull request by PerseidMeteor**
## Relevant Issue (if applicable)
https://github.com/dragonflyoss/nydus/issues/1596

## Details
Add `namespace` flag for `nydusify commit`.

## Types of changes

_What types of changes does your PullRequest introduce? Put an `x` in all the boxes that apply:_

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation Update (if none of the other choices apply)

## Checklist

_Go over all the following points, and put an `x` in all the boxes that apply._

- [x] I have updated the documentation accordingly.
- [x] I have added tests to cover my changes.

### CODE REVIEW

**Comment by PerseidMeteor** [comment]
PTAL :) @imeoer

File: contrib/nydusify/cmd/nydusify.go [hunk 1]
```diff
@@ -1179,6 +1179,12 @@ func main() {
 					Usage:   "Containerd address, optionally with \"unix://\" prefix [$CONTAINERD_ADDRESS] (default \"/run/containerd/containerd.sock\")",
 					EnvVars: []string{"CONTAINERD_ADDR"},
 				},
+				&cli.StringFlag{
+					Name:    "namespace",
```
Comment (imeoer) [inline contrib/nydusify/cmd/nydusify.go:1183]: Maybe also add an aliases `n`. :)

File: contrib/nydusify/pkg/committer/commiter.go [hunk 2]
```diff
@@ -84,7 +85,7 @@ func (cm *Committer) Commit(ctx context.Context, opt Opt) error {
 		return errors.Wrap(err, "parse target image name")
 	}
 
-	inspect, err := cm.manager.Inspect(ctx, opt.ContainerID)
+	inspect, err := cm.manager.Inspect(ctx, opt.ContainerID, opt.Namespace)
```
Comment (imeoer) [inline contrib/nydusify/pkg/committer/commiter.go:88]: How about calling `ctx = namespaces.WithNamespace(ctx, namespace)` before here.

File: contrib/nydusify/cmd/nydusify.go (hunk 1 above)
Comment (PerseidMeteor) [reply contrib/nydusify/cmd/nydusify.go:1183]: Add `aliases`.

File: contrib/nydusify/pkg/committer/commiter.go (hunk 2 above)
Comment (PerseidMeteor) [reply contrib/nydusify/pkg/committer/commiter.go:88]: How about calling in `nydusify.go` `main` func?
Just in `return cm.Commit(c.Context, opt)`

Comment (imeoer) [reply contrib/nydusify/pkg/committer/commiter.go:88]: Better to put it in `committer.go`, so we can import it as a package in other  project. :)

**Review by imeoer — APPROVED** [review]
thanks, LGTM!

**Comment by shusley244** [comment]
thanks~

**Pull request merged**